.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
```

Comment:
Not automaticly. But there are plugins for every editor/IDE, and this plugins are popular.
https://atom.io/packages/editorconfig